src/NServiceBus.Core.Tests/MessageMutators/MutateInstanceMessage/MutateOutgoingMessageBehaviorTests.cs

```diff
@@ -11,10 +11,26 @@
     [TestFixture]
     class MutateOutgoingMessageBehaviorTests
     {
+        [Test]
+        public async Task Should_invoke_all_explicit_mutators()
```

Comment:
do we miss tests which verify that we invoke both container registered and API registered mutators?